### PR TITLE
Clean cache prior to artifact build

### PR DESCRIPTION
**Pull request by JessterB**
This should significantly decrease the size of the zip artifact

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -41,6 +41,7 @@ jobs:
       node_js: 16
     - stage: eb-deploy-develop
       script:
+        - npm run clean:npm
```
Comment (zaro0508) [inline .travis.yml:44]: When I run this command then zip up files I get a file that 60MB to deploy to AWS.  Is this correct? have you tested this?
```
➜ zip /tmp/agora-app.zip -qr9 * .[^.]*
➜  ll /tmp/agora-app.zip
-rw-r--r--  1 zaro0508  wheel    60M Aug 15 09:42 /tmp/agora-app.zip
```

Comment (JessterB) [reply .travis.yml:44]: @zaro0508 The small zip generated via the changes in this PR has been manually deployed to the new dev environment, so this should be ready to merge.

Comment (zaro0508) [reply .travis.yml:44]: thanks.  i will test with aws EB CLI.

Comment (zaro0508) [reply .travis.yml:44]: This command fails with `sh: rimraf: command not found`.  Looks like an `npm install` to update dependencies will fix it though.  Could you please do that and commit the resulting `package-lock.json` file with this PR? 

```
➜  npm run clean:npm
> agora@3.0.0 clean:npm
> npm cache clean --force && npm run rimraf -- node_modules

npm WARN using --force Recommended protections disabled.

> agora@3.0.0 rimraf
> rimraf "node_modules"

sh: rimraf: command not found
```

**Review by zaro0508 — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 1 above)
Comment (mattcgenui) [reply .travis.yml:44]: @zaro0508 let me know if that's better

**Review by mattcgenui — APPROVED** [review]

**Comment by zaro0508** [comment]
I've built and deployed (exactly as how it's setup in travis.yaml) but my version of the artifact [failed to deploy](https://postimg.cc/7GyG9X3h).  My built artifact label is [app-v2020_07_23-681-g1c27-220816_144621921566-stage-220816_144621921712](https://postimg.cc/5H8jXvhV).

**Review by zaro0508 — CHANGES_REQUESTED** [review]
I've built and deployed (exactly as how it's setup in travis.yaml) but my version of the artifact [failed to deploy](https://postimg.cc/7GyG9X3h).  My built artifact label is [app-v2020_07_23-681-g1c27-220816_144621921566-stage-220816_144621921712](https://postimg.cc/5H8jXvhV).

**Pull request merged**